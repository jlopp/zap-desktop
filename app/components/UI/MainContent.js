import React from 'react'
import BackgroundDark from 'components/UI/BackgroundDark'

/**
 * @render react
 * @name MainContent
 * @example
 * <MainContent>Some content</MainContent>
 */
const MainContent = props => (
  <BackgroundDark as="article" width={1} p={3} css={{ height: '100%' }} {...props} />
)

export default MainContent
