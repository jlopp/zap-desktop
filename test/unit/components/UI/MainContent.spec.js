import React from 'react'
import MainContent from 'components/UI/MainContent'
import renderer from 'react-test-renderer'

describe('component.UI.MainContent', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<MainContent />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
