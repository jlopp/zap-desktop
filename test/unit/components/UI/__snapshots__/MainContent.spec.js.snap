// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component.UI.MainContent should render correctly 1`] = `
.c0 {
  padding: 16px;
  width: 100%;
  color: white;
  background-color: darkestBackground;
  height: 100%;
}

<article
  className="c0"
  color="white"
  width={1}
/>
`;
